src/common/wasm_vm.rs

```diff
@@ -28,14 +36,97 @@ pub enum PluginInstruction {
 pub struct PluginEnv {
     pub plugin_id: u32,
     // FIXME: This should be a big bundle of all of the channels
```

Comment:
Hey! We can finally get rid of this comment I left here!

Reply:
Yes - there are other instances of these comments I forgot to fix too!

Reply:
Think I got most of them...